cerebrium/fine-tuning/language-models/dataset.mdx

```diff
@@ -10,10 +10,10 @@ description: "Advice on how best to structure your datasets"
 </Note>
 
 The fine-tuning library has been built to leverage Parameter Efficient Fine-tuning and Low-Rank Approximation to reduce the number of trainable parameters by >99.9% while providing results that are comparable to full fine-tuning.  
-Due to the much lower number of trainable parameters, the datasets used do not need to be greater than 1000 examples however, it's always best to use as many, diverse training examples as possible to cover a wide variety of use cases.
+Due to the much lower number of trainable parameters, the datasets used do not need to be greater than 1000 examples. Often, much smaller datasets of 200-500 examples are more than sufficient, however, it's always best to use as many diverse training examples as possible to cover a wide variety of use cases.
```

Comment:
Often much smaller datasets of 200-500 examples are more than sufficient to achieve decent results. However, it's always best to use as many diverse training examples as possible to cover a wide variety of use cases.

Reply:
I like. Will do!